wodles/aws/aws_s3.py

```diff
@@ -1084,14 +1090,24 @@ def check_bucket(self):
             else:
                 print("ERROR: No files were found in '{0}'. No logs will be processed.".format(self.bucket_path))
                 exit(14)
-        except botocore.exceptions.ClientError as err:
-            if err.response['Error']['Code'] == 'ThrottlingException':
+
+        except botocore.exceptions.ClientError as error:
+            error_message = "Unknown"
+            exit_number = 1
+            error_code = error.response.get("Error", {}).get("Code")
+
+            if error_code == 'ThrottlingException':
```

Comment:
Adapt this code fragment to behave in the same way as the other ones you updated. Use `error_message` and `exit_number` for consistency.